src/build/dlopen.c

```diff
@@ -0,0 +1,130 @@
+/**
+ * @file dlopen.c
+ *
+ * @author James Jones (jejones@networkradius.com)
+ * @copyright 2020 Network RADIUS SARL (legal@networkradius.com)
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include <dlfcn.h>
+#include <gnumake.h>
+
+#ifdef __APPLE__
+#  define DL_EXTENSION ".dylib"
+#else
+#  include <linux/limits.h>
+#  define DL_EXTENSION ".so"
+#endif
+
+char		*mk_dlpath(char const *nm, unsigned int argc, char **argv);
```

Comment:
These functions can be `static`, as they don't need to be exported.

Reply:
True; will make that change.